### PR TITLE
Fix Docker file-passing test.

### PR DESCRIPTION
**Pull request by mcovarr**
Modify Docker file-passing test to pass deterministic content so assertions can be made to see if the files were actually passed!

### CODE REVIEW

File: src/test/scala/cromwell/MainSpec.scala [hunk 1]
```diff
@@ -5,10 +5,11 @@ import java.io.ByteArrayOutputStream
 import akka.actor.ActorSystem
 import akka.testkit.EventFilter
 import com.typesafe.config.ConfigFactory
+import cromwell.binding.WdlExpression
 import cromwell.server.WorkflowManagerSystem
 import cromwell.util.{SampleWdl, FileUtil}
 import cromwell.util.SampleWdl.ThreeStep
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
```
Comment (scottfrazer) [inline src/test/scala/cromwell/MainSpec.scala]: Bleh, I think I forgot to optimize the imports on this file after removing that function

File: src/main/scala/cromwell/engine/backend/local/LocalBackend.scala [hunk 2]
```diff
@@ -47,30 +47,27 @@ class LocalBackend extends Backend with LazyLogging {
...
+
     commandWriter.writeWithNewline(commandLine)
     commandWriter.flushAndClose()
 
     def buildDockerRunCommand(image: String): String =
       // -v maps the host workflow executions directory to /root/<workflow id> on the container.
-      // -w sets the command's working directory on the container to /root/<workflow id>/call-<call name>.
       // -i makes the run interactive, required for the cat and <&0 shenanigans that follow.
-      s"docker run -v ${hostWorkflowExecutionFile.getAbsolutePath}:$containerExecutionDir -w $containerExecutionDir/call-${call.name} -i $image"
+      s"docker run -v ${hostWorkflowExecutionFile.getAbsolutePath}:$containerExecutionDir -i $image"
```
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/local/LocalBackend.scala]: I'm not sure I understand this change

Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/local/LocalBackend.scala]: In the previous code, the non-Docker path would execute as the first line of the command script:

```
cd cromwell-executions/workflow-name/UUID/call-dir
```
while the Docker path would do the `-w /root/UUID/call-dir` to get similar behavior.  This change harmonizes the two paths to both do the cd in the command script.

**Comment by geoffjentry** [comment]
:+1: from me, not sure if you consider this ready for merge

**Comment by mcovarr** [comment]
I think this is mergeable, I don't feel bad about not adding tests to offsets the +5 increase in generated LOC for the parser fix.  :smile:

**Pull request merged**